### DISCUSSION TITLE
emergency freq

### DISCUSSION THREAD

**Discussion by TecnoTalarn** [foldynl/QLog#460] [Ideas]
![image](https://github.com/user-attachments/assets/419332e2-0df8-4c39-8094-10c4d2406a72)
Hi, an idea...
would it be possible to mark on the bandmap the frequencies that are recommended for emergency communications?
for example a line in red
![image](https://github.com/user-attachments/assets/88134a1d-5ec1-4b4a-a0a0-8d4a8e55ae0a)

**Comment by foldynl** [comment]
I think this issue is a bit more complex, but it's a good idea.

https://www.iaru-r1.org/about-us/committees-and-working-groups/emcomm/emergency-communications-frequencies/
